containers/java-postgres/test-project/.gitignore

```diff
@@ -0,0 +1 @@
+target
```

Comment:
How about adding a standard Java `.gitignore` file that lists non-code files, e.g. *.class.

See here for an example: https://github.com/github/gitignore/blob/main/Java.gitignore

Reply:
hi @xtophs, agree that could be done -- the current version came from the [java/test-project](https://github.com/microsoft/vscode-dev-containers/tree/main/containers/java/test-project) folder, and we didn't alter it

Reply:
We'll add the gitignore in. Small, QOL improvement.